packages/web/src/generic.ts

```diff
@@ -89,7 +89,11 @@ function inject(
  */
 function track(
   name: string,
-  properties?: Record<string, AllowedPropertyValues>
+  properties?: Record<string, AllowedPropertyValues>,
+  options?: {
+    flagKeys?: string[];
+    flags?: Record<string, boolean | number | string>;
```

Comment:
@tobiaslins you've added this here, but the script does not seem to support it. Should we remove it again?

Reply:
Yea, removed it